GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2267,9 +2269,18 @@ private void CommitMessageToolStripMenuItemDropDownOpening(object sender, EventA
         {
             var msg = AppSettings.LastCommitMessage;
             var maxCount = AppSettings.CommitDialogNumberOfPreviousMessages;
+            var authorPattern = "";
```

Comment:
```suggestion
            var authorPattern = string.Empty;
```

Reply:
Half the file uses ```""``` and the other ```string.Empty```. Should I apply a action to the whole file?

Reply:
This comment was for what you added